core/src/components/input/input.ionic.scss

```diff
@@ -103,25 +105,41 @@
   display: none;
 }
 
+// Input Label
+// ----------------------------------------------------------------
+
+.label-text-wrapper {
+  color: $ionic-color-neutral-700;
+
+  font-size: $ionic-font-size-s;
+  font-weight: $ionic-font-weight-medium;
+
+  line-height: $ionic-font-line-height-s;
+}
+
+:host(.label-floating) .label-text-wrapper {
+  @include transform(none);
```

Comment:
By removing this transform we are able to set the `font-size` directly instead of scaling it down from the `:host` font size.